### PR TITLE
Fix typo in analyses copies

### PR DESCRIPTION
**Pull request by elenatorro**

### CODE REVIEW

**Comment by Jesus89** [comment]
Just remember that the version should be bumped after the merge to avoid conflicts with other releases at the same time, as indicated by the Deploy doc :wink:

**Comment by elenatorro** [comment]
You are right, @Jesus89! 👌

File: NEWS.md [hunk 1]
```diff
@@ -152,13 +152,15 @@ ion for time-series (#12670)
...
 ### Bug fixes / enhancements
+* Fix typo in analyses copies (#13681)[https://github.com/CartoDB/cartodb/pull/13681]
 * Add helper text to mapbox basemap view (#13699)
 * Fix legends not refreshing when moving layers (#13696)
 * Fix broken api keys for organization users
 * Fix multiple bugs in widgets (#13686)
 * Fix category widget search on Android (https://github.com/CartoDB/support/issues/1074)
 * Improve pagination in category widgets (https://github.com/CartoDB/support/issues/1161)
 * Fix onboardings in layer content views (#13674)
+* Fix typo in analyses copies [#13681](https://github.com/CartoDB/cartodb/pull/13681)
```
Comment (ivanmalagon) [inline NEWS.md]: This is duplicated

Comment (elenatorro) [reply NEWS.md]: Fixed

**Pull request closed without merging**